src/angularComponent/ngcOmniboxComponent.js

```diff
@@ -0,0 +1,16 @@
+import NgcOmniboxController from './ngcOmniboxController.js';
+
+export default {
+  bindings: {
+    ngModel: '<',
+    ngDisabled: '&',
+    multiple: '<',
+    placeholder: '<',
+    accessoryText: '<',
+    autofocus: '<',
+    onShowSuggestions: '&',
+    onHideSuggestions: '&'
```

Comment:
I would like to see these bindings get added slowly over time instead of all at once in this first commit. Can you limit the bindings to just the ones that you're actually using in this commit?